css/widget.css

```
@@ -2,4 +2,9 @@
   background-color: white;
   height: 400px;
   width: 100%;
+  resize: both;
```

```suggestion
  resize: both;
  border: 1px solid white;
```

Not sure if this will be needed but adding border in here too might prevent flicker on hover.